site/mex_invenio/custom_facets.py

```diff
@@ -0,0 +1,37 @@
+# custom_facets.py
+from invenio_records_resources.services.records.facets import NestedTermsFacet
+
+class RestrictedTermsFacet(NestedTermsFacet):
```

Comment:
This filters the result set of a query, rather than filtering the initial query itself. This has the side effect of larger queries and incorrect counts.